docs/DeveloperGuide.md

```diff
@@ -308,7 +308,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case: Add a person**
+
+**MSS**
+1. User adds details of new person to be added to AddressBook.
+2. AddressBook adds new person with relevant details.
```

Comment:
Should you still be using 'AddressBook though'

Reply:
changed to TimeTable